code/game/objects/items/devices/radio/radio.dm

```diff
@@ -274,6 +274,10 @@ var/global/list/default_medbay_channels = list(
 	//  Fix for permacell radios, but kinda eh about actually fixing them.
 	if(!M || !message) return 0
 
+	if(speaking.flags & NONVERBAL) return 0
+
+	if(speaking.flags & SIGNLANG) return 0

```

Comment:
Bug on Bay, just being fixed now; these two need condensing into `if(speaking && (speaking.flags & (SIGNLANG|NONVERBAL)))`, or at the very least a check for `speaking` not being null. Right now AI radio speech runtimes if they don't have a default language set.